### ISSUE TITLE
Problem: REGISTRY and DOMAIN_NAME need to be defined when deploying

### ISSUE THREAD

**Issue by mamedin** [JiscRDSS/rdss-archivematica#219]
To deploy a local dev environment with shibboleth enabled, I need to run:

```
REGISTRY="localhost:5000/" DOMAIN_NAME=example.ac.uk make all ENV=dev SHIBBOLETH_CONFIG=archivematica
```
The `REGISTRY` var is needed to create the Shibboleth container. 

When the `DOMAIN_NAME` is not defined, the shibboleth service doesn't work.

**Comment by sevein** [comment]
@mamedin thank you so much, this worked for me too! 💯

**Comment by mamedin** [comment]
Using the `REGISTRY` variable has no sense when deploying with ENV=dev, it was a workaround to use a `shib-sp-proxy` image.

Using the DOMAIN_NAME was another workaround that forces to redeploy and to restart the services.

To fix the issue:

* `make all ENV=dev SHIBBOLETH_CONFIG=archivematica` command should restart all services at the end of the deploy.
* The shib-sp-proxy should use a context instead of a docker image when ENV=dev.

**Comment by sevein** [comment]
Closed by #222.
